stubs/requests/requests/structures.pyi

```diff
@@ -17,5 +18,5 @@ class LookupDict(dict[str, _VT]):
     name: Any
     def __init__(self, name: Any = ...) -> None: ...
     def __getitem__(self, key: str) -> _VT | None: ...  # type: ignore[override]
-    def __getattr__(self, attr: str) -> _VT: ...
     def __setattr__(self, __attr: str, __value: _VT) -> None: ...
+    def get(self, key: str, default: _D | None = ...) -> _VT | _D | None: ...  # type: ignore[override]
```

Comment:
Shouldn't this just have the same signature as `Mapping.get()` in `typing.pyi`? The only difference seems to be that the `key` parameter is positional-or-keyword, instead of being positional-only, as it is on `Mapping.get()`.

Reply:
I don't know about mapping, but whatever `dict` uses as it calls `self.__dict__.get(key, default)` and subclasses `dict` — I'll take a look at this shortly